SDK Showcase/SDK Showcase/Interactors/SDKInteractor.swift

```diff
@@ -194,3 +236,12 @@ private extension SDKInteractor {
         return ConfigModel(dictionary: model.dictionary)
     }
 }
+
+private class SimpleUserProfile: UserProfile {
```

Comment:
SimpleUserProfile? the name is unfortunate. How about ShowCaseProfile()

Pls also move it to a separate file

Reply:
sure